Problem1.py

```diff
@@ -0,0 +1,63 @@
+# Time Complexity :O(n^m
+# Space Complexity :O(n^m)
+#  Did this code successfully run on Leetcode :yes
+# Any problem you faced while coding this :no
```

Comment:
Good Job! Think on the space complexity. There is a small error. Let us know if you need further explanation.

Reply:
This would be O(1) I guess. Because we are returning that array